data/data/vsphere/main.tf

```diff
@@ -0,0 +1,77 @@
+provider "vsphere" {
+  user                 = var.vsphere_username
+  password             = var.vsphere_password
+  vsphere_server       = var.vsphere_url
+  allow_unverified_ssl = true
```

Comment:
hmm, this should not be allowed :(

Reply:
@abhinavdahiya unfortunately unless the vSphere admin provides a verified certificate there is no option to provide a certificate in the terraform provider.  Unless its added locally to the trust on the machine that `openshift-install` is ran - though I haven't tried that to see if it works.
https://www.terraform.io/docs/providers/vsphere/index.html#argument-reference

Reply:
My preference would be not allow trusted-skip. For installer the host should trust it, for cluster this is similar to the openstack API trust with self signed certs.

Let's make sure we track it somewhere.

Reply:
@abhinavdahiya the certificates are available for download.
```
wget --no-check-certificate  https://vcsa.vmware.devcluster.openshift.com/certs/download.zip
unzip download.zip
cd certs/lin
sudo cp * /etc/pki/ca-trust/source/anchors
sudo update-ca-trust extract
```
Do we want to require this and just document the steps above?

cc: 
@dav1x 
@mtnbikenc

Reply:
I just tested this.  Adding the certificates to the trust works as expected.